gh-release.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+lastReleasedVersion="$(gh release view -R crc-org/crc --json tagName --jq .tagName)"
```

Comment:
So this script add `gh` as dependency?

Reply:
yeah, alternatively we could directly talk to the github api using curl or something but with `gh` its much easier to do